integration/spark/src/main/scala/org/apache/spark/sql/optimizer/MVRewriteRule.scala

```diff
@@ -51,7 +53,11 @@ class MVRewriteRule(session: SparkSession) extends Rule[LogicalPlan] {
     } catch {
       case e =>
         // if exception is thrown while rewriting the query, will fallback to original query plan.
-        MVRewriteRule.LOGGER.warn("Failed to rewrite plan with mv: " + e.getMessage)
+        MVRewriteRule.LOGGER
```

Comment:
i think this change not required as we are just adding the message here, no need to again enable debug. please revert this change